bower.json

```diff
@@ -44,6 +44,7 @@
     "manageiq-ui-components": "0.0.12",
     "moment-strftime": "~0.2.0",
     "moment-timezone": "~0.4.1",
+    "moment-duration-format": "~1.3.0",
```

Comment:
You will need to do this change in ManageIQ/manageiq, the canonical version of `bower.json` lives there for now.